src/quo2/components/avatars/wallet_user_avatar/view.cljs

```diff
@@ -0,0 +1,56 @@
+(ns quo2.components.avatars.wallet-user-avatar.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.avatars.wallet-user-avatar.style :as style]
+            utils.string))
+
+(def circle-sizes
```

Comment:
personally I would do this as 
```
(def properties
  {:small   {:size 20
             :font-size :label
             :font-weight :medium }
   :medium {:size ...
            :font-size ...
            :font-weight ...}
   :large {:size ...
           :font-size ...
           :font-weight ...}
   :x-large {:size   ...
             :font-size ...
             :font-weight ...}})
```

as it becomes easier to see what properties are set for each size.

Reply:
additionally we have updated the approach to sizes. please see - https://github.com/status-im/status-mobile/pull/17279

Reply:
I agree with @J-Son89's approach. That's how it's handled these situations in the project